common/vendor/MarkdownEditor/angular-markdown-editor.js

```diff
@@ -179,25 +179,63 @@ function addNewButtons() {
   //       }]
   // },
   {
-        name: "groupPrev",
-        data: [{
+      name: "groupUtil",

```

Comment:
Moved this to be next to the inline review button. I think it looks a lot better and probably something no one noticed.